docker/supabase/volumes/db/init/04-initiate-tables.sql

```diff
@@ -0,0 +1,14 @@
+CREATE extension IF NOT EXISTS citext  WITH SCHEMA extensions;
+
+CREATE TABLE IF NOT EXISTS telescope_profiles (
+  id text PRIMARY KEY,
+  registered_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
+  updated_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
+  feeds text[] NOT NULL,
+  display_name citext UNIQUE NOT NULL,
+  first_name text NOT NULL,
+  last_name text NOT NULL,
+  email citext UNIQUE NOT NULL,
+  github_username citext UNIQUE NOT NULL,
+  github_avatar_url citext UNIQUE NOT NULL
```

Comment:
Do you think we need to store this, since we can construct it from the user's GitHub username?

Also, I'm not sure about case insensitive values for a URL.

Reply:
GitHub uses user id for avatars now, @humphd's avatar  https://avatars.githubusercontent.com/u/427398?v=4

Reply:
Changed the URL from citext to text.